api/dp-openapi.yaml

```diff
@@ -591,6 +591,354 @@ paths:
         default:
           $ref: '#/components/responses/UnexpectedError'
 
+  /projects/badpagedata:
+    get:
+      tags:
+      - project
+      description: Gets data for mark bad page
+      responses:
+        200:
+          description: Data
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/bad_page_data'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '404':
```

Comment:
is this possible?

Reply:
Probably not. I just copied all the error responses at the end of each  endpoint.